pkg/backup/safe_point.go

```diff
@@ -83,16 +93,12 @@ func StartServiceSafePointKeeper(
 	}
 	updateTick := time.NewTicker(updateGapTime)
 	checkTick := time.NewTicker(checkGCSafePointGapTime)
+	update(ctx)
 	go func() {
 		defer updateTick.Stop()
 		defer checkTick.Stop()
 		for {
 			select {
-			case <-ctx.Done():
-				// Before finish backup, we have to make sure
-				// the backup ts does not fall behind with GC safepoint.
-				check(context.TODO())
-				return
 			case <-updateTick.C:
```

Comment:
hmm so the loop never ends until the process exits? this may pose a problem to the embedded BR in TiDB.

Reply:
Oops... I just want to remove the `check` because the it will always fail since client is closed. However some strange power make me remove the full select branch. 😅